book/content/open-source-comms/comms-channels.md

```diff
@@ -0,0 +1,34 @@
+# Communication Channels
+
+The channel(s) through which you choose to communicate with your community will have a big impact on the experience and sense of belonging for your community members.
```

Comment:
Maybe:

Communication within a community impacts the contributor experience, influences an individual's sense of belonging, and provides a historical record of project decisions.